modules/administration-guide/partials/proc_configuring-a-che-workspace-with-a-persistent-volume-strategy.adoc

```diff
@@ -116,10 +116,10 @@ $ {orch-cli} apply -f _<my-cluster.yaml>_
 [subs="+quotes,+attributes"]
 ----
 $ {orch-cli} patch checluster {prod-checluster} --type=json \
-  -p '[{"op": "replace", "path": "/spec/storage/pvcStrategy", "value": "__<per-workspace>__"}]'
+  -p '[{"op": "replace", "path": "/spec/storage/pvcStrategy", "value": "__per-workspace__"}]'
```

Comment:
I am not using the operator installer, so I'll need a second opinion on this.  Is `per-workspace` fine in this case, and is the `/spec/storage/pvcStrategy` path the right one in the CRD `checluster`?